### ISSUE TITLE
REST API Products filter not working as expected

### ISSUE THREAD

**Issue by alexzandros** [akeneo/pim-community-dev#11412]
**I'm reporting a Bug**
[akeneo-pim-system-info_2020-01-15_07_38.txt](https://github.com/akeneo/pim-community-dev/files/4065666/akeneo-pim-system-info_2020-01-15_07_38.txt)

**Expected Behavior**
When I use the REST API and try to create a filter for products that have certain value on any given attribute that has a numeric id like the following:
```GET /api/rest/v1/products?search={"1602":[{"operator":"=","value":"hua"}]} HTTP/1.1```
I expect to receive the pagination results as happens when the attribute has a text Id

**Current Behavior**
The shown query returns a 500 response with the following json body
``` json
{
    "code": 500,
    "message": "Internal Server Error"
}
```

**Steps to Reproduce**
- Create an attribute with an id of 1000
- Assign that attribute a value in any given set of products
- Use the REST API filter facility to filter products by that said attribute

**Comment by Doodoune** [comment]
Hi @alexzandros,

Could you send us the logs in you file var/logs/prod.log that correspond to this error 500?
It's more verbose and it will be easier to understand what is going wrong.
Regards,

**Comment by alexzandros** [comment]
Hello @Doodoune,

 I'm sending the logs requested

Thanks,
[prod.zip](https://github.com/akeneo/pim-community-dev/files/4067129/prod.zip)

**Comment by mmetayer** [comment]
Hello @alexzandros,

The issue was fixed in #11420, and the fix is available in the last tag (v3.2.32).  Have a nice day!
